rfcs/text/0001-rfc-process.md

```diff
@@ -211,6 +273,46 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
   ***@channel**: DCP PM declined the RFC for the RFC Process - https://github.com/HumanCellAtlas/dcp-community/pull/27*
 
+**ADDITION BEGINS**
+### Pausing RFCs
+
+#### Author(s):
+
+- If via various discussions, it is decided that the RFC must be "paused" until further information can be obtained, and rough consensus has been met to pause rather than reject or withdraw, replace "rfc-community-review" with "rfc-paused" and comment in the PR with the rationale.
+
+#### Shepherd:
+
+- Announce the pausing of the RFC on the HumanCellAtlas #dcp slack channel. Include a link to the RFC pull request:
+
+  **EXAMPLE**
+
+  ***@channel**: DCP PM has decided to pause the RFC in the RFC Process - https://github.com/HumanCellAtlas/dcp-community/pull/27*
+  
+### Withdrawing RFCs
+
+#### Author(s):
+
+- Authors may decide to withdraw an RFC at any point in time. The difference between a withdrawl and a rejection is that the rejected RFC has fully completed the review period and addressed all comment with the rough consensus being a rejection whereas a withdrawl has not yet fully completed the review process. Replace "rfc-community-review" with "rfc-withdrawn" and close the pull request with the rationale in the comment.
+
+#### Shepherd:
+
+- Announce the withdrawl of the RFC on the HumanCellAtlas #dcp slack channel. Include a link to the RFC pull request:
+
+  **EXAMPLE**
+
+  ***@channel**: Authors have withdrawn the RFC for the RFC Process - https://github.com/HumanCellAtlas/dcp-community/pull/27*
+
+### Deprecating RFCs
+
+#### Author(s):
+
```

Comment:
For editorial consistency with the original RFC, _label_ should be used instead of _tag_.

Reply:
Done.